agent-install/agent-install.sh

```diff
@@ -3307,6 +3309,18 @@ function get_cluster_image_arch() {
     echo $image_arch
 }
 
+# check if the storage class exists in the edge cluster
+function check_cluster_storage_class() {
+    log_debug "check_cluster_storage_class() begin"
+    local storage_class=$1
+    if $KUBECTL get storageclass ${storage_class} >/dev/null 2>&1; then
+        log_verbose "storage class $storage_class exists in the edge cluster"
+    else
+        log_fatal 2 "$storage_class does not exist in the edge cluster"
```

Comment:
Can you check this log_fatal message to 
```
log_fatal 2 "storage class $storage_class does not exist in the edge cluster"
```

Currently it fails with
```
2024-07-18 14:26:10 KUBECTL is set to oc
2024-07-18 14:26:10 EDGE_CLUSTER_STORAGE_CLASS: fred (from environment variable)
2024-07-18 14:26:10 ERROR: fred does not exist in the edge cluster
```

I guess it is a little obvious that it is checking for the storage class but this would make it very clear